tests/test_download.py

```diff
@@ -129,6 +131,27 @@ def test_no_credentials_error(self, remote_local_file: Any):
             download_from_gcs(mock_remote_filepath, mock_local_filepath)
 
 
+class TestDatabricksUnityCatalog:
+
+    def test_invalid_prefix_from_db_uc(self, remote_local_file: Any):
+        with tempfile.NamedTemporaryFile(delete=True, suffix='.txt') as tmp:
+            file_name = tmp.name.split(os.sep)[-1]
+            mock_remote_filepath, _ = remote_local_file(cloud_prefix='dbfs:/Volumess',
+                                                        filename=file_name)
+            with pytest.raises(Exception, match='Expected path prefix to be.*'):
+                download_from_databricks_unity_catalog(mock_remote_filepath, tmp.name)
+
+
+class TestDatabricksFileSystem:
+
+    def test_invalid_prefix_from_dbfs(self, remote_local_file: Any):
+        with tempfile.NamedTemporaryFile(delete=True, suffix='.txt') as tmp:
+            file_name = tmp.name.split(os.sep)[-1]
+            mock_remote_filepath, _ = remote_local_file(cloud_prefix='dbfsx:/', filename=file_name)
```

Comment:
should `cloud_prefix` here be `'dbfs:/'` without the 'x'?

Reply:
I intentionally kept it to hit the exception in case use made such a mistake, then we should throw a proper descriptive error message.